docs/source/api/layers/index.rst

```diff
@@ -0,0 +1,18 @@
+`layers`
+=========
+
+Layers implements ready to use `neural network` layers that come with crypto for free.
```

Comment:
Is there another way we can write this other than with crypto for free? A user may not know what "crypto for free" means? 

cc @mortendahl